spring-boot/admin/src/main/java/eu/coatrack/admin/service/GatewayConfigFilesStorage.java

```diff
@@ -0,0 +1,104 @@
+package eu.coatrack.admin.service;
+
+/*-
+ * #%L
+ * coatrack-admin
+ * %%
+ * Copyright (C) 2013 - 2020 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import eu.coatrack.api.Proxy;
+import eu.coatrack.api.ServiceApi;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+/**
+ *
+ * @author perezdf, ChristophBaier
+ */
+
+@Slf4j
+@Component
+public class GatewayConfigFilesStorage {
+
+    @Value("${proxy.config.files.folder}")
+    private String gatewayConfigFilesFolderLocation;
+
+    @Value("${ygg.admin.api-base-url-for-gateway}")
+    private String adminApiBaseUrlForGateway;
+
+    @Value("${is-run-via-docker-compose-setup}")
+    private boolean isDockerComposeSetup;
+
+    @PostConstruct
+    private void emptyProxyConfigFilesFolderIfExistsForNonDockerComposeDeployments() throws IOException {
+        if (!isDockerComposeSetup){
+            Path targetFolderPath = Paths.get(System.getProperty("user.dir") + "/target");
+            if (Files.notExists(targetFolderPath))
+                Files.createDirectory(targetFolderPath);
+
+            Path gatewayConfigFilesFolderPath = Paths.get(gatewayConfigFilesFolderLocation);
+            if (Files.exists(gatewayConfigFilesFolderPath)){
+                FileUtils.deleteDirectory(gatewayConfigFilesFolderPath.toFile());
```

Comment:
If I understand correctly, CoatRack deletes something from my hard drive on every start-up. Does not seem like a good idea to me, as it could in some situations delete important stuff by accident. To be discussed.